usr/lib/common/aix/asprintf.h

```diff
@@ -0,0 +1,52 @@
+#ifndef UTIL_LINUX_ASPRINTF_H
```

Comment:
Where does this implementation come from? If you wrote it yourself, then this file needs a copyright and license header, otherwise we should include the original header....

Reply:
We do need a copyright/license header no matter what. We can't have an unlicensed source file.